src/main/java/core/mate/academy/service/producerimpl/BulldozerProducerImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service.producerimpl;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class BulldozerProducerImpl implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        IntStream.range(1, 10)
```

Comment:
the usage of `IntStream` for simple iteration is unjustified

comparing to a simple `for` loop, stream requires much more resources and gives no benefits in this case

**please apply the same to other implementations**